tests/cases/draw_triangle_list_with_depth.vkscript

```diff
@@ -43,44 +43,40 @@ void main() {
 
 #             Red for entire frame
 #        R8   G8 B8  A8     R8   G8  B8 A8
-       -128 -128  3  127    255   0   0  0
-        127  127  3  127    255   0   0  0
-       -128  127  3  127    255   0   0  0
-
-       -128 -128  3  127    255   0   0  0
-        127  127  3  127    255   0   0  0
-        127 -128  3  127    255   0   0  0
+       -128 -128  96  127    255   0   0  0
+        127  127  96  127    255   0   0  0
+       -128  127  96  127    255   0   0  0
+       -128 -128  96  127    255   0   0  0
+        127  127  96  127    255   0   0  0
+        127 -128  96  127    255   0   0  0
 
 #             Green for half frame
-          0 -128  2  127      0 255   0  0
-        127  127  2  127      0 255   0  0
-          0  127  2  127      0 255   0  0
-
-          0 -128  2  127      0 255   0  0
-        127  127  2  127      0 255   0  0
-        127 -128  2  127      0 255   0  0
+          0 -128  64  127      0 255   0  0
+        127  127  64  127      0 255   0  0
+          0  127  64  127      0 255   0  0
+          0 -128  64  127      0 255   0  0
+        127  127  64  127      0 255   0  0
+        127 -128  64  127      0 255   0  0
 
 #           Blue for quarter frame
-       -128    0  1  127      0   0 255   0
-          0  127  1  127      0   0 255   0
-       -128  127  1  127      0   0 255   0
-
-       -128    0  1  127      0   0 255   0
-          0  127  1  127      0   0 255   0
-          0    0  1  127      0   0 255   0
+       -128    0  32  127      0   0 255   0
+          0  127  32  127      0   0 255   0
+       -128  127  32  127      0   0 255   0
+       -128    0  32  127      0   0 255   0
+          0  127  32  127      0   0 255   0
+          0    0  32  127      0   0 255   0
 
 #         Mixed color for quarter frame
-          0    0  0  127    127 127 127   0
-        127  127  0  127    127 127 127   0
-          0  127  0  127    127 127 127   0
-
-          0    0  0  127    127 127 127   0
-        127  127  0  127    127 127 127   0
-        127    0  0  127    127 127 127   0
+          0    0  16  127    127 127 127   0
```

Comment:
Ditto, why bump up the blues?

Reply:
this was part an attempt to figure out why this test does not work in dawn. I'll revert it.

Reply:
The blues here are being used as Z values.
We wanted to make sure there was no flakiness with precision of the depth values.  So we separated them by wide intervals.   This might not have been strictly necessary, we did this while trying to debug the depth flakiness in the Dawn flow.